src/Compilers/CSharp/Test/Emit/Emit/CompilationEmitTests.cs

```diff
@@ -4984,5 +5000,28 @@ public override void Method(T input)
                 //     public override void Method(T input)
                 Diagnostic(ErrorCode.ERR_CantChangeReturnTypeOnOverride, "Method").WithArguments("DerivingClass<T>.Method(T)", "BaseClass<T>.Method(T)", "int").WithLocation(11, 26));
         }
+
+        [Fact]
+        public void CompileAndVerifyModuleInlucesAllModules()
+        {
+            // Before this change, CompileAndVerify() didn't include other modules when testing a PEModule.
+            // Verify that symbols from other modules are accessible as well.
+
+            var modRef = ModuleMetadata.CreateFromImage(CreateStandardCompilation("public class A { }", options: TestOptions.ReleaseModule, assemblyName: "refMod").EmitToArray());
```

Comment:
>EmitToArray [](start = 164, length = 11)

I think we have a helper EmitToImageReference or something similar that gives a ready to use reference.